db/change_cache_test.go

```diff
@@ -279,14 +279,16 @@ func TestLateSequenceErrorRecovery(t *testing.T) {
 	// Returns slice of non-nil events received.
 	nextFeedIteration := func() []*ChangeEntry {
 		events := make([]*ChangeEntry, 0)
+		ticker := time.NewTicker(10 * time.Second)
```

Comment:
Let's not bother changing any test code. There are some slight behaviour differences between time.After, time.Ticker, time.Timer which would need a closer look at.

Reply:
Reverted changes from all test code.